app/src/main/java/org/p2p/wallet/rpc/repository/solana/RpcSolanaRemoteRepository.kt

```diff
@@ -28,6 +29,8 @@ class RpcSolanaRemoteRepository(
     private val gson: Gson
 ) : RpcSolanaRepository {
 
+    private var epochInfoCache: EpochInfo? = null
```

Comment:
check that RpcSolanaRemoteRepo is singleton

Reply:
it's definitely singleton
```
single {
            val api = get<Retrofit>(named(RPC_RETROFIT_QUALIFIER)).create(RpcSolanaApi::class.java)
            RpcSolanaRemoteRepository(api, get(), get())
        } bind RpcSolanaRepository::class
```